### DISCUSSION TITLE
Rename FluentPaginator in FluentPagination

### DISCUSSION THREAD

**Discussion by franklupo** [microsoft/fluentui-blazor#2292] [General]
HI,
What do you think about renaming Fluent Paginator to Fluent Pagination?
I've seen other libraries call it Pagination.

**Comment by dvoituron** [comment]
Each library has its own name: 
- Telerik uses `Paging` and `GridPagerSettings`
- MudBlazor uses `MudPagination`
- Radzen uses `AllowPaging`

And we are using `FluentPaginator` 😊

What's the benefit of changing the name and forcing hundreds of thousands of developers to adapt the code?
